.github/workflows/release.yaml

```diff
@@ -14,11 +14,22 @@ jobs:
     environment: Changesets
     runs-on: ubuntu-latest
     steps:
+      - name: Use Lit Robot's name and email for all git actions
+        run: |
+          git config --global user.name "Lit Robot"
+          git config --global user.email "lit-robot@google.com"
+
       - name: Setup Node.js 16
         uses: actions/setup-node@v2
         with:
           node-version: 16
 
+      # changesets/action@v1 currently has a bug where it attempts to set the
+      # local git user info before changing directories for `cwd`, which means
+      # it will fail if the root directory is not a git repo.
```

Comment:
May be worth adding a TODO to a Lit issue or changesets issue since this is a workaround comment. To track cleaning this up.

Reply:
This is now #2542.